test/evergreen.yml

```diff
@@ -1870,33 +1881,25 @@ tasks:
           script: |
             set -o errexit
             set -o verbose
-            # Archive the WT_TEST directories which include the generated wt data files
+            # Archive the WT_TEST directories which include the generated wt data files. We cannot
+            # use the Evergreen archive command as we need to archive multiple WT_TEST folders.
             tar -zcvf WT_TEST.tgz WT_TEST*
-      - command: s3.put
-        params:
-          aws_secret: ${aws_secret}
-          aws_key: ${aws_key}
+      - func: "upload endian format artifact"
+        vars:
+          endian_format: big-endian
           local_file: wiredtiger/build_posix/test/format/WT_TEST.tgz
-          bucket: build_external
-          permissions: public-read
-          content_type: application/tar
-          display_name: WT_TEST
-          remote_file: wiredtiger/big-endian/${revision}/artifacts/WT_TEST.tgz
+          remote_file: WT_TEST.tgz
```

Comment:
Can we reflect the endianness format in the file name e.g. `WT_TEST-${endian_format}.tgz`?

Reply:
See 5141f0800a9fabe9b0ee7c867a0969aff5d57242